src/pages/v2/p/components/V2Dashboard.tsx

```diff
@@ -165,7 +165,9 @@ export default function V2Dashboard({
     nftRewardsCIDs = CIDsOfNftRewardTiersResponse(nftRewardTiersResponse)
   }
   const { data: nftRewardTiers, isLoading: nftRewardTiersLoading } =
-    useNftRewards(nftRewardTiersResponse ?? [])
+    nftRewardsCIDs
+      ? useNftRewards(nftRewardTiersResponse ?? [])
+      : { data: undefined, isLoading: false }
```

Comment:
Gets rid of the `allTiers` error on non-NFT projects (closes #1690)

Reply:
hmm. hooks shouldnt be called conditionally (React should actually be erroring in this case, or eslint).

I'm guessing the below should be working but isnt. Maybe the fix is there instead?

https://github.com/jbx-protocol/juice-interface/blob/main/src/hooks/v2/NftRewards.ts

Reply:
Ah true for sure